src/plugins/controls/timer/plugin.js

```diff
@@ -177,6 +193,7 @@ export default pluginFactory({
                     .then(function(startZen) {
                         //set up the timerbox
                         self.timerbox = timerboxFactory({
+                            ariaHidden: isReviewPanelEnabled(),
```

Comment:
This does not work if the review panel is enabled later.
You should also do something  after `renderitem`: https://github.com/oat-sa/tao-test-runner-qti-fe/pull/158/files#diff-80a5ac2a02cffb57a0fcff319627f957R181

For instance, naive approach: `self.timerbox.getElement().attr('aria-hidden', isReviewPanelEnabled());`